examples/discovery-search-app/src/setupProxy.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const dotenv = require('dotenv');
+const serverEnv = dotenv.config({ path: path.join(__dirname, '../', '.server-env') });
+const clusterEnv = dotenv.config({ path: path.join(__dirname, '../', '.env.local') });
+const proxy = require('http-proxy-middleware');
+const { CloudPakForDataAuthenticator } = require('ibm-watson/auth');
+const cors = require('cors');
+const bodyParser = require('body-parser');
+
+if (serverEnv.error) {
+  console.warn(serverEnv.error);
+}
+
+if (clusterEnv.error) {
+  console.warn(clusterEnv.error);
+}
+
+const RELEASE_PATH = process.env.RELEASE_PATH;
+const BASE_URL = process.env.BASE_URL;
+const addAuthorization = async (req, _res, next) => {
+  const authenticator = new CloudPakForDataAuthenticator({
+    url: BASE_URL || 'http://example.com',
+    username: process.env.CLUSTER_USERNAME || 'username',
+    password: process.env.CLUSTER_PASSWORD || 'password',
+    disableSslVerification: true
+  });
+  try {
+    const accessToken = await authenticator.tokenManager.getToken();
+    req.headers.authorization = `Bearer ${accessToken}`;
+  } catch (e) {
+    console.error(e);
+  }
+  return next();
```

Comment:
what is the `next` param here?

Reply:
just says move to the next step in middleware